app/models/collection.ts

```diff
@@ -39,6 +39,9 @@ export default class CollectionModel extends OsfModel {
     @hasMany('registration', { inverse: null })
     linkedRegistrations!: DS.PromiseManyArray<RegistrationModel>;
 
+    @hasMany('collected-metadatum', { inverse: 'collection' })
+    collectedMetadata!: DS.PromiseManyArray<CollectedMetadatumModel>;
+
```

Comment:
Added a `collected-metadatum` relationship on our API/v2 endpoint. See the other PR.